backend/src/main/conf/backend-conf.json

```diff
@@ -2,16 +2,16 @@
   "ip.address": "vmnathost",
   "backend.version": 1,
   "backend.id": 1,
-  "backend.http.port": 2493,
-  "leader.http.port": 2498,
   "load.report.interval.secs": 10,
   "recorder.defunct.threshold.secs": 30,
   "slot.pool.capacity": 100,
   "backend.http.server": {
-    "idle.timeout.secs": 120
+    "port": 2491,
+    "idleTimeout": 120
```

Comment:
Let's follow one convention for json fields, foo.bar.baz or fooBarBaz. Opinions?